tests/utils/datavolume.go

```diff
@@ -105,6 +105,33 @@ func NewDataVolumeWithHTTPImport(dataVolumeName string, size string, httpURL str
 	}
 }
 
+// NewDataVolumeWithHTTPImportToBlockPV initializes a DataVolume struct with HTTP annotations to import to block PV
+func NewDataVolumeWithHTTPImportToBlockPV(dataVolumeName string, size string, httpURL string) *cdiv1.DataVolume {
+	dataVolume := &cdiv1.DataVolume{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: dataVolumeName,
+		},
+		Spec: cdiv1.DataVolumeSpec{
+			Source: cdiv1.DataVolumeSource{
+				HTTP: &cdiv1.DataVolumeSourceHTTP{
+					URL: httpURL,
+				},
+			},
+			PVC: &k8sv1.PersistentVolumeClaimSpec{
+				AccessModes: []k8sv1.PersistentVolumeAccessMode{k8sv1.ReadWriteOnce},
+				Resources: k8sv1.ResourceRequirements{
+					Requests: k8sv1.ResourceList{
+						k8sv1.ResourceName(k8sv1.ResourceStorage): resource.MustParse(size),
+					},
+				},
+			},
```

Comment:
it complains: "cannot take the address of "kubevirt.io/containerized-data-importer/vendor/k8s.io/api/core/v1".PersistentVolumeMo....."

Reply:
Hmm corev1.PersistentVolumeBlock is a const and you should be able to take a pointer to it like this:
```go
PVC: &k8sv1.PersistentVolumeClaimSpec{
	AccessModes: []k8sv1.PersistentVolumeAccessMode{k8sv1.ReadWriteOnce},
	Resources: k8sv1.ResourceRequirements{
		Requests: k8sv1.ResourceList{
			k8sv1.ResourceName(k8sv1.ResourceStorage): resource.MustParse(size),
		},
	},
        VolumeMode: &corev1.PersistentVolumeBlock
},
```

Reply:
Since the above doesn't work, can you at least do:
```go
volumeMode := k8sv1.PersistentVolumeBlock
...
PVC: &k8sv1.PersistentVolumeClaimSpec{
    AccessModes: []k8sv1.PersistentVolumeAccessMode{k8sv1.ReadWriteOnce},
        Resources: k8sv1.ResourceRequirements{
            Requests: k8sv1.ResourceList{
		k8sv1.ResourceName(k8sv1.ResourceStorage): resource.MustParse(size),
            },
	},
        VolumeMode: &volumeMode
    },


```